airflow/contrib/operators/awsbatch_operator.py

```diff
@@ -110,10 +115,17 @@ def execute(self, context):
         )
 
         try:
+            if 'memory' in self.overrides:
```

Comment:
Does the client check the types? If so, I would rather let client validate the input and raise error.